packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -106,6 +110,7 @@ export class Relayer {
           relayTransactions.push(await this._generateRelayTransactionForPendingDeposit(l1Token, relayableDeposit));
         } catch (error) {
           this.logger.error({ at: "AcrossRelayer#Relayer", message: "Unexpected error processing deposit", error });
+          console.log("ERROR", error);
```

Comment:
nit console.log

Reply:
ye, good catch. this was an artifact of my painful debugging with some flaky unit tests! have removed.